composer.json

```diff
@@ -11,6 +11,9 @@
     ],
     "require": {
         "php": ">=7.4",
+        "ext-bcmath": "*",
+        "ext-gmp": "*",
+        "ext-json": "*",
```

Comment:
Why `"*"` all of these? Shouldn't we define the versions?

Reply:
External dependencies in composer are commonly used just as an "check whether the tool is installed and enabled" if you don't need any specific or new feature.